### PR TITLE
Tensorboard instructions (generic)

### PR DESCRIPTION
**Pull request by erskordi**

### CODE REVIEW

**Review by KevinSayers — CHANGES_REQUESTED** [review]
Just one update otherwise thing it looks good.

File: MachineLearning/TensorBoard/tensorboard.md
Comment (erskordi) [reply MachineLearning/TensorBoard/tensorboard.md:34]: Oh yes, you're right! I focused on the directory for `ray` only, but you're right, it has to be more generic. I will change it asap!

File: MachineLearning/TensorBoard/tensorboard.md
Comment (KevinSayers) [inline MachineLearning/TensorBoard/tensorboard.md:29]: Sounds good. I suggest dropping the Conda bit too just to keep it generic except the tensorboard portion. Once that is removed can merge.

**Review by KevinSayers — CHANGES_REQUESTED** [review]
One more small change.

File: MachineLearning/TensorBoard/tensorboard.md
Comment (erskordi) [reply MachineLearning/TensorBoard/tensorboard.md:29]: The thing is that to use Tensorboard, you need to activate the Conda environment (at least that's how I know it, unless one would install Tensorboard at the `base` env)

Comment (erskordi) [reply MachineLearning/TensorBoard/tensorboard.md:29]: What do you think?

Comment (KevinSayers) [reply MachineLearning/TensorBoard/tensorboard.md:29]: You could use it from a pip install in userspace or in a container images as well. I think it is best to keep it as 'agnostic' to the install as possible. I'll defer to you though if you think it is important we can leave it in and merge as is.

Comment (erskordi) [reply MachineLearning/TensorBoard/tensorboard.md:29]: Oooh I see, I didn't know that you can use alternative methods like that to activate Tensorboard. In that case, I could rephrase it as:

```There are three main methods for activating Tensorboard:
* If you included Tensorboard installation in an Anaconda environment, activate it and proceed
* You can also install Tensorboard in userspace using `pip install`
* Install it using container images

Then [...]
```

P.S. is it possible you provide me with examples for the pip install userspace or container images so that I include them in the tutorial as well?

Comment (KevinSayers) [reply MachineLearning/TensorBoard/tensorboard.md:29]: Below is an example of running it from a container
```
ml singularity-container
singularity pull docker://tensorflow/tensorflow
singularity run tensorflow_latest.sif
tensorboard --logdir=. --port 6006 --bind_all
```

Python user
```
pip install tensorboard --user
tensorboard --logdir=. --port 6006 --bind_all
```
Yes having the different options listed like you have would be good.

Comment (erskordi) [reply MachineLearning/TensorBoard/tensorboard.md:29]: Great, thanks! I will perform the changes asap. When done, take a look and lmk if it's ok

**Pull request merged**
